### PR TITLE
Copy updates

### PR DESCRIPTION
**Pull request by OnFreund**

### CODE REVIEW

File: steps/03_run_both_servers.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 id: run_both_servers
 learningObjectives:
-  - Ensure both the Python and Node.js servers run together using Docker.
+  - Ensure both the Python and Node.js servers are running together using Docker.
```
Comment (OnFreund) [inline steps/03_run_both_servers.yml]: Is this the real objective though?

**Comment by OnFreund** [comment]
Additionally, steps 3-5 instruct the user to create a PR - why aren't we validating them?

File: steps/03_run_both_servers.yml (hunk 1 above)
Comment (shem8) [reply steps/03_run_both_servers.yml]: Probably not

Comment (OnFreund) [reply steps/03_run_both_servers.yml]: I think we've been abusing the field for a while now. Maybe we should only have quest level objectives?

Comment (shem8) [reply steps/03_run_both_servers.yml]: No every step should have learning objectives, that's ok

**Comment by shem8** [comment]
> Additionally, steps 3-5 instruct the user to create a PR - why aren't we validating them?

This is the first version, once we'll approve it with them we'll add the checks

**Pull request merged**